source/slang/slang-ast-base.h

```diff
@@ -514,6 +703,10 @@ class Decl : public DeclBase
         SLANG_RELEASE_ASSERT(state >= checkState.getState());
         checkState.setState(state);
     }
+
+private:
+    SLANG_UNREFLECTED DeclRefBase* m_defaultDeclRef = nullptr;
+    SLANG_UNREFLECTED Index m_defaultDeclRefEpoch = -1;
```

Comment:
I may have scanned past it all, but are there comments that would help somebody new to the code understand how these "epochs" work?